autopilot/README.md

```diff
@@ -0,0 +1,12 @@
+# Autopilot package.
+The autopilot consists of control loops that actuate on setpoints that have been set by the mission control.
+
+These setpoints are:
+- **Attitude Servo**. Roll, pitch, yaw and speed targets.
+- **Altitude Heading**. Altitude, heading and speed targets.
+- **Depth Heading**. Depth, heading and speed targets.
+- **Fix Rudder**. Rudder, depth and speed targets.
+- **Waypoint**. Altitude, latitude, longitude and speed targets.
```

Comment:
@patapufate meta: I wonder if we should mention topics instead. I'm a bit divided.

Reply:
I think we should leave it as setpoints. We are describing "functions" instead of "implementation".